cookbook/how_to_install_extension_from_command_line.rst

```diff
@@ -0,0 +1,58 @@
+.. index::
+    single: Extension; Install extension
+    single: Extension; Install extension from command line
+    single: Extension
+
+How to install Oro Platform extensions
+======================================
+
+*Used application: OroPlatform 1.**
+
+There are several ways to install extensions from command line:
+
+.. caution::
+
+    Before extension installation we recommend to backup database and application code source. Once installed extension cannot be uninstalled
```

Comment:
I would reword this a bit:

> Before installing an extension it is recommended to back up the database and the application source code. There is no simple way to uninstall an extension.

I also think that it is better to move this before the sentence in line 11.